kong/plugins/request-termination/handler.lua

```diff
@@ -15,7 +15,7 @@ local RequestTerminationHandler = {}
 
 
 RequestTerminationHandler.PRIORITY = 2
-RequestTerminationHandler.VERSION = "2.0.1"
+RequestTerminationHandler.VERSION = "2.1.0"
```

Comment:
Same here for the request termination plugin.

Reply:
(commit removed)